README.md

```diff
@@ -138,6 +138,11 @@ gladly accepted.
 
 - row  [like in a table]
   col
+  
+- fst  [1st, 2nd, 3rd, 4th]
+  snd
+  trd
+  fth
```

Comment:
This is confusing wrt. fifth

Reply:
Yes. admittedly only `fst` and `snd` exist in the Haskell prelude.

Reply:
I'm not sure about this suggestion as these abbreviations are pretty confusing (as @danr points out) and not commonplace---I've never run into `trd` or `fth`.

It's quite *odd* that 
```
first
third
fifth
```

line up *even* though
```
zeroth
fourth
second
```

Reply:
Some nice puns and coincidences there :) closing this PR since these names are too confusing apparently